### PR TITLE
Correct element access of Map

### PR DESCRIPTION
**Pull request by nmtigor**
`this._actions` is assigned by `createActionsMap()` which returns Map.

### CODE REVIEW

**Comment by Snuffleupagus** [comment]
(Again, a slightly better commit message wouldn't hurt here.)

This seems like the sort of change that really needs a test-case, to both ensure that it works correctly now and in order to prevent any future regressions.

**Review by Snuffleupagus — COMMENTED** [review]
@calixteman Can you perhaps help out with creating a test-case here, and any further reviewing necessary, since I don't think we should land this without a test?

File: src/scripting_api/field.js [hunk 1]
```diff
@@ -457,9 +457,9 @@ class Field extends PDFObject {
       return;
     }
     if (!(cTrigger in this._actions)) {
```
[the remark this answers is not in the document]
Comment (calixteman) [reply src/scripting_api/field.js:459]: Yep, it's totally wrong...
Fortunately, I think `setAction` isn't used that much.
@nmtigor, are you able to create a very basic pdf where we could change an action on a field ?

**Comment by Snuffleupagus** [comment]
The updated patch doesn't address (any of) the comments above, and the commit message also looks like it was intended for another PR.

File: src/scripting_api/field.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nmtigor) [reply src/scripting_api/field.js:459]: No. It's complicated for me currently.

**Pull request closed without merging**